.github/workflows/release.yaml

```diff
@@ -3,104 +3,99 @@ name: release
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.release.outputs.upload_url }}
+          asset_path: release/ubuntu-18.04/ironoxide-cli-ubuntu-18.04.asc
+          asset_name: ironoxide-cli-ubuntu-18.04.asc
+          asset_content_type: application/pgp-signature
 
-    - name: Download release artifacts from macos-10.15
-      uses: actions/download-artifact@v1
-      with:
-        name: release-macos-10.15
-        path: release/macos-10.15
-    - name: Sign artifact for macos-10.15
-      run: |
-        gpg --batch --detach-sign -a release/macos-10.15/ironoxide-cli-macos-10.15
-        gpg --batch --verify release/macos-10.15/ironoxide-cli-macos-10.15.asc release/macos-10.15/ironoxide-cli-macos-10.15
-    - name: Upload artifact for macos-10.15
-      uses: actions/upload-release-asset@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        upload_url: ${{ steps.release.outputs.upload_url }}
-        asset_path: release/macos-10.15/ironoxide-cli-macos-10.15
-        asset_name: ironoxide-cli-macos-10.15
-        asset_content_type: application/data
-    - name: Upload signature for macos-10.15
-      uses: actions/upload-release-asset@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        upload_url: ${{ steps.release.outputs.upload_url }}
-        asset_path: release/macos-10.15/ironoxide-cli-macos-10.15.asc
-        asset_name: ironoxide-cli-macos-10.15.asc
-        asset_content_type: application/pgp-signature
+      - name: Download release artifacts from macos-10.15
+        uses: actions/download-artifact@v1
+        with:
+          name: release-macos-10.15
+          path: release/macos-10.15
+      - name: Sign artifact for macos-10.15
+        run: |
+          gpg --batch --detach-sign -a release/macos-10.15/ironoxide-cli-macos-10.15
+          gpg --batch --verify release/macos-10.15/ironoxide-cli-macos-10.15.asc release/macos-10.15/ironoxide-cli-macos-10.15
+      - name: Upload artifact for macos-10.15
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.release.outputs.upload_url }}
+          asset_path: release/macos-10.15/ironoxide-cli-macos-10.15
+          asset_name: ironoxide-cli-macos-10.15
+          asset_content_type: application/data
+      - name: Upload signature for macos-10.15
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.release.outputs.upload_url }}
+          asset_path: release/macos-10.15/ironoxide-cli-macos-10.15.asc
+          asset_name: ironoxide-cli-macos-10.15.asc
+          asset_content_type: application/pgp-signature
 
   build:
     strategy:
       matrix:
-        os: [ ubuntu-18.04, macos-10.15 ]
+        os: [ubuntu-18.04, macos-10.15, windows-2019]
```

Comment:
You should probably add logic to the `release` job, above, to download and publish the artifacts for Windows.

Reply:
Good catch, I completely missed that step.